drivers/volume/portworx/portworx.go

```diff
@@ -1004,25 +1082,21 @@ func (p *portworx) localSnapshotRestore(
 	}
 	for volID, snapID := range restoreVolumes {
 		logrus.Infof("Restoring volume %v with local snapshot %v", volID, snapID)
-
+		if isCloudSnap {
+			snapID = restoreNamePrefix + volID
+		}
 		err = volDriver.Restore(volID, snapID)
 		if err != nil {
 			logrus.Errorf("Unable to restore volume %v with ID %v, err %v",
 				volID, snapID, err)
 			return err
 		}
 		logrus.Infof("Completed restore for volume %v with Snapshotshot %v", volID, snapID)
+		// TODO: Delete restore objects
```

Comment:
@disrani-px  should we delete restore object here ? or have separate api to cleanup restore objects.
So handleDelete will also delete pending restore objects
Restore objects here are - 
volumes with prefix `in-place-restore`

Reply:
Once the restore is complete to the original volume the temporary volume should be deleted. The restore should not be marked succesful until that is deleted otherwise it could leave orphaned volumes